docs/configuration/duo-push-notifications.md

```diff
@@ -24,6 +24,7 @@ duo_api:
   hostname: api-123456789.example.com
   integration_key: ABCDEF
   secret_key: 1234567890abcdefghifjkl
+  enable_self_enrollment: true
```

Comment:
```suggestion
  enable_self_enrollment: false
```

Generally I try to keep the docs example the same as the template, thoughts?

Reply:
Agreed.